cfgov/unprocessed/css/organisms/header.less

```diff
@@ -83,6 +86,18 @@
             & > .m-global-search {
                 float: right;
             }
+
```

Comment:
> Instead of nesting could you use the m-global-header-cta__horizontal to float the CTA?

You mean put `float: right;` in `m-global-header-cta__horizontal`? So the rationale not to is to encapsulate the molecule. Having the float in the molecule means it would be aware of its layout. We should be able to move the molecule around without worrying about floats blowing up the layout of other molecules. Does that make sense?

Note: there are floats on the mega-menu trigger and global search. Added a todo for those in https://github.com/cfpb/cfgov-refresh/commit/f3bec89f514f05e43e86189e7949256d9616c598

> Does Search even need the nesting? There's only one.

I merged it into the `.m-global-header-cta` rule, does this work? https://github.com/cfpb/cfgov-refresh/commit/f3bec89f514f05e43e86189e7949256d9616c598